src/site/_includes/authors-index.njk

```diff
@@ -1,26 +1,36 @@
 ---
-layout: default
+layout: default-next
 ---
 {% from 'macros/author-card.njk' import authorCard with context %}
-<div>
-  {# Make sure Actions are first in the tab order. #}
-  <div class="w-actions">
+
+<div class="authors">
```

Comment:
is this class used somewhere?

Reply:
It is not, I think I added it just to give it context. I'll remove uses like this.